### PR TITLE
created bulk_scan_orchestrator with HASHCODE permission

### PR DESCRIPTION
**Pull request by kaanaktas**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CCD-2125

### Change description ###

bulk_scan_orchestrator has been created with HASHCODE permission

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/service_config.json [hunk 1]
```diff
@@ -127,6 +127,14 @@
         "CREATE",
         "READ"
       ]
+    },
+    {
+      "id": "bulk_scan_orchestrator",
+      "caseTypeId": ["*"],
+      "jurisdictionId": "*",
+      "permissions": [
+        "HASHTOKEN"
```
[the remark this answers is not in the document]
Comment (mdayican) [reply src/main/resources/service_config.json:135]: @kaanaktas @smathangi I missed in the ticket but we need CREATE access for bulk_scan_orchestrator for functional tests. I can update the ticket or create new one.

[the remark this answers is not in the document]
Comment (smathangi) [reply src/main/resources/service_config.json:135]: @mdayican why can't you use "bulk_scan_processor" S2S token in your functional tests ?

[the remark this answers is not in the document]
Comment (mdayican) [reply src/main/resources/service_config.json:135]: i'd prefer not to mix them if you can give permissions per env.

**Review by smathangi — APPROVED** [review]

File: src/main/resources/service_config.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MSancaktutar) [reply src/main/resources/service_config.json:135]: @mdayican  - I would kindly suggest we don't provide access to only satisfy functional tests unless the access is already how it should ideally be. On that basis is it ok if we release the PR as is at the moment?

[the remark this answers is not in the document]
Comment (mdayican) [reply src/main/resources/service_config.json:135]: ok.

**Review by smathangi — APPROVED** [review]

**Pull request merged**